### PR TITLE
fabric8 --> jkube for spring-rhoar-intro

### PR DESCRIPTION
**Pull request by dewan-ahmed**

### CODE REVIEW

File: spring/spring-rhoar-intro/pom.xml [hunk 1]
```diff
@@ -38,13 +38,12 @@
...
     <spring-boot-bom.version>2.1.6.Final-redhat-00004</spring-boot-bom.version>
     <maven-surefire-plugin.version>2.20</maven-surefire-plugin.version>
-    <fabric8-maven-plugin.version>3.5.40</fabric8-maven-plugin.version>
-    <fabric8.openshift.trimImageInContainerSpec>true</fabric8.openshift.trimImageInContainerSpec>
-    <fabric8.skip.build.pom>true</fabric8.skip.build.pom>
-    <fabric8.generator.from>
+<jkube.openshift.trimImageInContainerSpec>true</jkube.openshift.trimImageInContainerSpec>
+    <jkube.skip.build.pom>true</jkube.skip.build.pom>
+    <jkube.generator.from>
       registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift:${openjdk18-openshift.version}
```
Comment (edeandrea) [inline spring/spring-rhoar-intro/pom.xml:44]: This needs to be changed to registry.redhat.io/ubi8/openjdk-11:latest

**Review by edeandrea — COMMENTED** [review]
See inline comments

**Review by edeandrea — COMMENTED** [review]
Why is the postgresql version needed? It should be brought in automatically by the bom.

**Comment by dewan-ahmed** [comment]
> Why is the postgresql version needed? It should be brought in automatically by the bom.

Initially I left out the postgresql version. The `mvn oc:deploy` execution fails with

```
$ mvn package oc:deploy -Popenshift -DskipTests
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for org.postgresql:postgresql:jar must be a valid version but is '${postgresql.version}'. @ line 146, column 20
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project com.example:spring-getting-started:1.0.0-SNAPSHOT (/root/rhoar-getting-started/spring/spring-rhoar-intro/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for org.postgresql:postgresql:jar must be a valid version but is '${postgresql.version}'. @ line 146, column 20
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```

That's why I had to add the postgresql version.

**Comment by edeandrea** [comment]
Instead of adding the `<postgresql.version>` element in `<properties>` remove thr `<version>` element within the `<dependency`> for the postgres dependency, like in https://github.com/openshift-katacoda/rhoar-getting-started/pull/62/commits/444bc8587e268a8a14a4fdfb74d82ff287082931#diff-765084dfd66ada00d467dcf49f5b7b3a4e9db65298304bf0249d8fdfdfb9d6e9L126

**Comment by dewan-ahmed** [comment]
@edeandrea I fixed what you suggested [here](https://github.com/openshift-katacoda/rhoar-getting-started/pull/58#issuecomment-864298187) and testing the scenario.

**Pull request merged**